### DISCUSSION TITLE
Would this "prompt blending" be possible to implement?

### DISCUSSION THREAD

**Discussion by 23pennies** [vladmandic/sdnext#2606] [General]
https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/1273
A year ago, someone made this PR for a new feature called "prompt blending", but it got denied. Still, that script was fully functional for a long time and it was a great little tool for me. But my auto1111 install broke, and with new installs it doesn't work anymore, and it would need to be adapted for SDXL (and diffusers?) anyway.
Does this look doable for SD.Next? I miss this feature a lot, but I also don't want to request something that would require a full rewrite for the prompt parser or something lol. I tried my hand at adapting the original to SDXL and got way over my head already.

**Comment by vladmandic** [comment]
difficult as prompt blending relies on interpolation during prompt scheduling and sdxl does not like prompt scheduling.
its possible in theory, but it would likely also come with a significant performance cost since prompt embeddings would need to be recalculated on each step. in general, its cool, but i don't consider it essential - such complicated prompting tools end up being used by very few ppl and don't make an overall impact.
